File: Student.java
class Student{
  private String name;
  private int age;
    private String major;

  public Student(String name, String major, int age){
    this.name = name;
    this.major = major;
    this.age = age;
  }

  public String getMajor(){
    return major;
  }

   public int getAge(){
    return age;
  }

  @Override
   public String toString(){
    return "Student{name:'"+name"', age:'"+age"', major:'"+major"'}";
  }

  public static void main(String[] args){
    Student student = new Student("John Doe","Software Engineering", 23);
    System.out.println(student);
  }
   
}

File: Student.py
"""
Module containing student entity classes.
"""

class Student:
    """Represents a student with name, age, and major."""

    def __init__(self, name: str, age: int, major: str) -> None:
        """
        Initialize a new Student.

        Args:
            name: The student's name
            age: The student's age in years
            major: The student's academic major
        """
        self.name = name
        self.age = age
        self.major = major

    def get_name(self) -> str:
        """Return the student's name."""
        return self.name

    def get_age(self) -> int:
        """Return the student's age."""
        return self.age

    def get_major(self) -> str:
        """Return the student's major."""
        return self.major

    def __str__(self) -> str:
        """Return string representation of the student."""
        return f'Student{{name="{self.name}", age={self.age}, major="{self.major}"}}'

    @staticmethod
    def main() -> None:
        """Demonstration method."""
        student_age = 23
        student = Student(
            "John Doe",
            student_age,
            "Software Engineering"
        )
        print(student)


if __name__ == "__main__":
    Student.main()
